src/PrestaShopBundle/ApiPlatform/Processor/CommandProcessor.php

```diff
@@ -31,32 +31,41 @@
 use ApiPlatform\Metadata\Operation;
 use ApiPlatform\State\ProcessorInterface;
 use PrestaShop\PrestaShop\Core\CommandBus\CommandBusInterface;
+use PrestaShopBundle\ApiPlatform\Exception\NoExtraPropertiesFoundException;
+use Symfony\Component\Serializer\Serializer;
 
 class CommandProcessor implements ProcessorInterface
 {
-    /**
-     * @var CommandBusInterface
-     */
-    private $commandBus;
```

Comment:
Moving the parameter from explicitely declared variable to constructor-declared parameter, and readonly, is a BC Break. Please document it in your PR description.